docs/HowToInstall.md

```diff
@@ -0,0 +1,90 @@
+# How to install?
+
+## Prerequisit
+
+ONE-vscode is an [extension](https://code.visualstudio.com/docs/editor/extension-marketplace) of [vscode](https://code.visualstudio.com/). To use ONE-vscode, you should install vscode first.
+
+### vscode and its extensions
+
+Let's install vscode according to [doc](https://code.visualstudio.com/docs/setup/setup-overview). (You should also install npm for vscode.)
+
+### ssh extension in vscode
+
+Now [ONE](https://github.com/Samsung/ONE) officially releases on linux/ubuntu only. If your development environment is on Windows, you should use ssh extension in vscode. Let's follow [tutorial](https://code.visualstudio.com/docs/remote/ssh-tutorial).
+
+### workspace
+
+Prepare workspace according to the [document](https://code.visualstudio.com/docs/editor/workspaces). ONE-vscode has been developed based on workspace.
+Typically, a workspace for ONE-vscode has the following files:
+- your model files
+  - Tensoflow model file(*.pb, saved model or Keras model)
+  - Tensorflow Lite model (*.tflite) 
+  - Onnx model (*.onnx)
+- ONE *.cfg files
+- intermediate files during compilation (e.g., *.circle) or compiled output files after compilation
+
+## How to install?
+
+If you are a user not a developer, installing ONE-vscode is enough. Let's install ONE-vscode with vsix file.
```

Comment:
(Q) What is a difference between `a user` and `a developer` in this sentence?

Reply:
My point was
- a user: just uses ONE-vscode
- a developer: developed ONE-vscode